libraries/AdaptiveExpressions/ExpressionProperties/ExpressionProperty.cs

```diff
@@ -70,19 +70,19 @@ public virtual Expression ToExpression()
 
             if (this.ExpressionText != null)
             {
-                this.expression = Expression.Parse(this.ExpressionText.TrimStart('='));
+                this.expression = new ExpressionParser().Parse(this.ExpressionText.TrimStart('='));
```

Comment:
>new ExpressionParser() [](start = 34, length = 22)

Please put Expression.Parse back across these files.  It reduces the concept count. (You only need to think about Expression class unless you want to understand the backing stuff.)

Reply:
yes, make sense, let's do that